osrf/mbzirc
# Add camera to Oberon7 arm and force-torque sensors to gripper

Added camera and force torque sensors to arm and gripper for manipulation task

To test:

1. launch the simple demo world and spawn an usv with arm and gripper

    ```
    # launch simple demo world
    ros2 launch ros_ign_gazebo ign_gazebo.launch.py ign_args:="-v 4 simple_demo.sdf"

    # spawn usv with arm and gripper
    ros2 launch mbzirc_ign spawn.launch.py name:=usv world:=simple_demo model:=usv type:=usv x:=15 y:=0 z:=0.3 R:=0 P:=0 Y:=0 arm:=mbzirc_oberon7_arm gripper:=mbzirc_oberon7_gripper
    ```

1. Run `ros2 topic list` to see the new topics:

    ```
    /usv/arm/gripper/joint/finger_left/wrench
    /usv/arm/gripper/joint/finger_right/wrench
    ...
    /usv/arm/wrist_link/camera_info
    /usv/arm/wrist_link/image_raw
    /usv/arm/wrist_link/optical/camera_info
    /usv/arm/wrist_link/optical/image_raw
    ```

1. Try echo a wrench topic: 

    ```
    ros2 topic echo /usv/arm/gripper/joint/finger_left/wrench
   ```

1. Run `rqt` or `rviz` to see the arm camera images:

    ```
    ros2 run rqt_image_view rqt_image_view
    ```
    and select the `/usv/arm/wrist_link/image_raw` topic



Signed-off-by: Ian Chen <ichen@osrfoundation.org>
